src/jit/flowgraph.cpp

```diff
@@ -16569,6 +16569,15 @@ void Compiler::fgExtendEHRegionBefore(BasicBlock* block)
 #endif // DEBUG
             HBtab->ebdTryBeg = bPrev;
             bPrev->bbFlags |= BBF_TRY_BEG | BBF_DONT_REMOVE | BBF_HAS_LABEL;
+#if FEATURE_EH_FUNCLETS
+            if (fgFuncletsCreated && ((block->bbFlags & BBF_FUNCLET_BEG) != 0))
+            {
+                bPrev->bbFlags |= BBF_FUNCLET_BEG;
+                block->bbFlags &= ~BBF_FUNCLET_BEG;
+            }
+#endif // FEATURE_EH_FUNCLETS
+
+
```

Comment:
Seems like this isn't necessary, because if `block` is the beginning of a funclet, then it will also be the first block of a handler, and thus the handler check below will do the right thing. Plus, if you do this, I would think the handler check assert below would fail because you would have already cleared the bit.